.github/workflows/gitlab.yml

```diff
@@ -31,4 +31,4 @@ jobs:
           project-id: ${{ secrets.PROJECT_ID }}
           schedule: ${{ github.event_name == 'schedule' }}
           cancel-outdated-pipelines: ${{ github.ref_name != 'main' }}
-          triggered-ref: v2.2.2 # REMEMBER to also update in .gitlab-ci.yml
+          triggered-ref: lukasp/batching-tests-long
```

Comment:
This will be reverted back, but for now it allows me to trigger the gitlab pipeline as the test contains `batching` pytest mark.

Reply:
at the moment, becasue of pytest.mark it will not be executed alongside regular libtelio tests and will not delay those unecessarily